bundles/org.openhab.binding.freeathomesystem/README.md

```diff
@@ -0,0 +1,81 @@
+
+|Version|Supported|
+|---|---|
+|2.6.1+|yes|
+
+
+# Setup / Installation
+
+## Prerequisites
+
+To make use of this Binding first the local free@home API has to be activated. The API is disabled by default!
+
+1. Open the free@home next app
+2. Browse to "Settings -> free@home settings -> local API and activate the checkbox
+
+## Discovery
+
+The free@home bridge shall be added manually. Once it is added as a Thing with correct credentials, the scan of free@home devices will be possible.
+The devices with multiple channels are devided into multiple devices with a single channel. The label of these devices is built-up with the same deivce name and free@home device ID but the channel number is mentioned in the Thing label as well for better identification.
+
+## Setup
+
+1. Enter your openHAB webfrontend with `<device IP>:8080`
+2. Log into openHAB with your crendetials at the lower left side
+3. Browse to "Settings -> Things" and press the "+" symbol
+4. Choose "FreeAtHome System Binding" and click "Free@home Bridge"
+5. Add the required data: SysAP IP address, username and password
+**ATTENTION:** The username here has to be from "Settings -> free@home settings -> local API, NOT the username from webfrontend or used in the app for login)
+
+6. Press save in the righter upper corner
+7. If everthing is right the Bridge should went "Online"
+8. "Scan" for the free@home devices and set up the your free@home components as Thing
+
+## free@home components as openHAB Things
+
+The free@home system is calling its components as free@home devices. The devices in the free@home system can have one or multiple channels depending the device's setup. During the scanning process the openHAB binding for free@home will split-up devices with multiple channels into multiple openHAB Things with single channel. However the name of the openHAB Thing will reflect the channel of the free@home device for simplier identification
+
+Example for a free@home Switch Actuator Sensor 2/2, which is having two actuator channles.
+
+|Device in free@home| |Things in openHAB|
+|---|---|---|
+|Device Actuator Sensor 2/2 - Name `Actuator_Livingroom`  | ⇨ |1st Thing - Name `Actuator_Livingroom_1`<br />2nd Thing - Name `Actuator_Livingroom_2`|
+
+## Sensors and Actuators of free@home devices as Things in openHAB
+
+The free@home system is supporting the sensors and actuators. the connenction of sensors and actuatators are done on the free@home system dashboard. The sensors in the free@home system are not accessible via free@home API (at the time of the development of this binding). Therefore the sensor only devices or the sensor channel(s) of a combined sensor/actuator devices are not represented in the openHAB binding.
+
+# Communities
+
+[Busch-Jaeger Community](https://community.busch-jaeger.de/)
+
+[free@home user group Facebook DE](https://www.facebook.com/groups/738242583015188)
+
+[free@home user group Facebook EN](https://www.facebook.com/groups/452502972031360)
```

Comment:
The "Full example" section with examples of .items and .things files is missing.

Reply:
### **Things File Example**
The binding is automatically detects the device and channel configuration and stores this information in the non-editable properties of the new Things. The Thing is using this information to access the free@home device through the bridge.

The channel configuration can be changed in the free@home dashboard, and the information of actual device and channel configuration is not visible on the free@home dashboard. It is true that there are some combination what the user can try, but a manual configuration is not useful. 

Therefore I made this design decision to make automatic configuration.

However the manual configuration would be possible, but in this case an access to the properties from the Things file must be made.
![Képernyőfotó 2022-02-06 - 17 09 50](https://user-images.githubusercontent.com/47272385/152690574-c9150bde-97a1-4659-9044-00fed0ba2a5c.png)

### **Items File Example**
Items example is added

Reply:
There are many users, who use textual configuration. That's why official bindings need to be configureable via text files. This is also possible with Things with dynamic Channels as suggested in the other comment.

The properties you implemented seem rather configuration parameters. Properties are intended to show additional information like the firmware version of the Thing. If a parameter is mandatory to make the Thing work, it is propably a configuration parameter.

Reply:
due to the re-written binding with dynamic thing and channel creation, the thing file example will be re-worked as well. this image will be removed in the next days